server/webapp/WEB-INF/rails/webpack/views/components/forms/input_fields.tsx

```diff
@@ -202,9 +205,16 @@ class HelpText extends MithrilViewComponent<HelpTextComponentAttrs> {
     }
   }
 
+  static maybeDocLink(docLink?: string) {
+    if (!_.isEmpty(docLink)) {
+      return <span>&nbsp;<Link href={docsUrl(docLink)} target="_blank">More&#8230;</Link></span>;
```

Comment:
See https://www.owasp.org/index.php/Reverse_Tabnabbing. Perhaps make the `Link` component smart enough, based on the `href` attribute to add additional attrs?

Reply:
Thanks for pointing that out. About the `Link` component, do you mean checking if the link specifies `target` and then adding `rel="noopener noreferrer"` if needed?

Reply:
What I meant is to make the `Link` component smart enough to specifically detect external links (`http(s)://`) and, based on that, automatically add the `target="_blank"` and `rel="noopener noreferrer"` attributes.